CODEOWNERS

```diff
@@ -1,482 +1,483 @@
+/bundles/org.openhab.binding.monopriceaudio/ @mlobstein
+/bundles/org.openhab.binding.mpd/ @stefanroellin
+/bundles/org.openhab.binding.mqtt/ @ccutrer
+/bundles/org.openhab.binding.mqtt.espmilighthub/ @Skinah
 /bundles/org.openhab.binding.mqtt.fpp/ @computergeek1507
-/bundles/org.openhab.binding.mqtt.generic/ @ccutrer
-/bundles/org.openhab.binding.mqtt.homeassistant/ @antroids @ccutrer
-/bundles/org.openhab.binding.mqtt.homie/ @ccutrer
-/bundles/org.openhab.binding.mqtt.ruuvigateway/ @ssalonen
-/bundles/org.openhab.binding.mycroft/ @dalgwen
-/bundles/org.openhab.binding.mybmw/ @ntruchsess @mherwege @martingrassl
-/bundles/org.openhab.binding.mynice/ @clinique
-/bundles/org.openhab.binding.mystrom/ @pail23
-/bundles/org.openhab.binding.nanoleaf/ @stefan-hoehn
-/bundles/org.openhab.binding.neato/ @jjlauterbach
-/bundles/org.openhab.binding.neeo/ @morph166955
-/bundles/org.openhab.binding.neohub/ @andrewfg
-/bundles/org.openhab.binding.nest/ @wborn
-/bundles/org.openhab.binding.netatmo/ @clinique @lolodomo
-/bundles/org.openhab.binding.network/ @mettke
-/bundles/org.openhab.binding.networkupstools/ @Hilbrand
-/bundles/org.openhab.binding.nibeheatpump/ @paulianttila
-/bundles/org.openhab.binding.nikobus/ @crnjan
-/bundles/org.openhab.binding.nikohomecontrol/ @mherwege
-/bundles/org.openhab.binding.nobohub/ @espenaf
-/bundles/org.openhab.binding.novafinedust/ @t2000
-/bundles/org.openhab.binding.ntp/ @marcelrv
-/bundles/org.openhab.binding.nuki/ @janvyb
-/bundles/org.openhab.binding.nuvo/ @mlobstein
-/bundles/org.openhab.binding.nzwateralerts/ @cossey
-/bundles/org.openhab.binding.oceanic/ @kgoderis
-/bundles/org.openhab.binding.ojelectronics/ @EvilPingu
-/bundles/org.openhab.binding.omnikinverter/ @hansbogert
-/bundles/org.openhab.binding.omnilink/ @ecdye
-/bundles/org.openhab.binding.onebusaway/ @sdwilsh
-/bundles/org.openhab.binding.onewire/ @J-N-K
-/bundles/org.openhab.binding.onewiregpio/ @aogorek
-/bundles/org.openhab.binding.onkyo/ @pail23 @paulianttila
-/bundles/org.openhab.binding.opengarage/ @psmedley
-/bundles/org.openhab.binding.opensprinkler/ @CrackerStealth @FlorianSW
-/bundles/org.openhab.binding.openthermgateway/ @ArjenKorevaar @andrewfg
-/bundles/org.openhab.binding.openuv/ @clinique
-/bundles/org.openhab.binding.openweathermap/ @cweitkamp
-/bundles/org.openhab.binding.openwebnet/ @mvalla
-/bundles/org.openhab.binding.oppo/ @mlobstein
-/bundles/org.openhab.binding.orbitbhyve/ @octa22
-/bundles/org.openhab.binding.orvibo/ @tavalin
+/bundles/org.openhab.binding.mqtt.generic/ @ccutrer
+/bundles/org.openhab.binding.mqtt.homeassistant/ @antroids @ccutrer
+/bundles/org.openhab.binding.mqtt.homie/ @ccutrer
+/bundles/org.openhab.binding.mqtt.ruuvigateway/ @ssalonen
+/bundles/org.openhab.binding.mycroft/ @dalgwen
+/bundles/org.openhab.binding.mybmw/ @ntruchsess @mherwege @martingrassl
+/bundles/org.openhab.binding.mynice/ @clinique
+/bundles/org.openhab.binding.mystrom/ @pail23
+/bundles/org.openhab.binding.myuplink/ @alexf2015

```

Comment:
Could you revet all changes to this file except this one?

Reply:
I do not understand this diff. I only added my line. It seems that line breaks were converted. But this is the only change. I will revert it.